tensorflow.md

```diff
@@ -1,32 +1,77 @@
-# Building TensorFlow on Graviton2
+# ML inference on Graviton CPUs with TensorFlow
 
-Install bazel build system:
+There are two ways to use tensforflow based ml inferencing on Graviton CPUs.
+- Tensorflow DLAMI
+- Tensorflow Docker container
+
+**Using AWS DLAMI:**
+To launch a Graviton instance with a DLAMI via EC2 console:
+
+1. Click on "Community AMIs"
```

Comment:
Putting the same is great, but how about providing a mechanism to find the latest AMI:
```
aws --profile admin ec2 describe-images --region us-east-1 --filters  Name=architecture,Values=arm64 Name=name,Values="*Deep Learning AMI*" Name=owner-alias,Values=amazon  --query 'Images[] | sort_by(@, &CreationDate)[-1] | ImageId'
```

Reply:
thanks, Ali. will add these details.